school/teacher.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'person'
+class Teacher < Person
+  def initialize(specialization)
+    @specialization = specialization
+    super()
+  end
```

Comment:
- [ ] As I've mentioned previously please make sure to update the parameters for the `Teacher` constructor here as well, it should have all the parameters in the parent constructor and only add `specialization`.
- [ ] Also make sure to include all the proper arguments when calling the parent class constructor in the `super()` at line `5`.